src/main/java/MisterDuke.java

```diff
@@ -0,0 +1,678 @@
+import java.io.*;
```

Comment:
Agreed with IceBear. Would also make program easier to edit rather than having to Crtl-F the class every time.

Also, can consider explicitly listing which packages are being imported rather than using wildcards.